package.json

```diff
@@ -87,31 +35,117 @@
+        },
+        "globalNodePath": {
+          "title": "Global Node Installation Path",
+          "description": "Write the value of `npm get prefix` here",
+          "type": "string",
+          "default": "",
+          "order": 2
+        }
       }
     },
-    "rulesToDisableWhileFixing": {
-      "title": "Disable specific rules from fixes",
-      "description": "Prevent rules from being auto-fixed by ESLint. Applies to fixes made during saves as well as when running the `Linter Eslint: Fix File` command.",
-      "type": "array",
-      "default": [],
-      "items": {
-        "type": "string"
+    "disabling": {
+      "type": "object",
+      "order": 6,
+      "properties": {
+        "disableWhenNoEslintConfig": {
+          "title": "Disable when no ESLint config is found (in package.json or .eslintrc)",
+          "type": "boolean",
+          "default": true,
+          "order": 1
+        },
+        "ignoreFixableRulesWhileTyping": {
+          "title": "Ignore fixable rules while typing",
+          "description": "Have the linter ignore all fixable rules during linting when editing a document. The list is automatically updated on each lint job, and requires at least one run to be populated. Only supported when using ESLint v4+.",
+          "type": "boolean",
+          "default": false,
+          "order": 2
+        },
+        "rulesToDisableWhileFixing": {
+          "title": "Disable specific rules from fixes",
+          "description": "Prevent rules from being auto-fixed by ESLint. Applies to fixes made during saves as well as when running the `Linter Eslint: Fix File` command.",
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "string"
+          },
+          "order": 3
+        },
+        "rulesToSilenceWhileTyping": {
+          "title": "Silence specific rules while typing",
+          "description": "Useful when Atom fixes errors on save like `no-trailing-spaces` or `eol-last`.",
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "string"
+          },
+          "order": 4
+        }
       }
     },
-    "ignoreFixableRulesWhileTyping": {
-      "title": "Ignore fixable rules while typing",
-      "description": "Have the linter ignore all fixable rules during linting when editing a document. The list is automatically updated on each lint job, and requires at least one run to be populated. Only supported when using ESLint v4+.",
-      "type": "boolean",
-      "default": false
+    "advanced": {
+      "type": "object",
+      "order": 7,
+      "properties": {
+        "disableEslintIgnore": {
+          "title": "Don't use .eslintignore files",
+          "type": "boolean",
+          "default": false,
+          "order": 1
+        },
+        "disableFSCache": {
+          "title": "Disable FileSystem Cache",
+          "description": "Paths of node_modules, .eslintignore and others are cached",
```

Comment:
I know you didn't change the wording here, but since we are touching it we might as well update a few things. Can you change this to "... others are normally cached"? The way it's currently worded the _description_ seems to indicate enabling this option enables caching, when the title (and behavior) do the exact opposite.

Reply:
I was planning to take another pass at the descriptions after this PR, I think there are a few things that could probably be clarified.  I'll make this change now though since you've pointed it out and we may as well.